### DISCUSSION TITLE
Github Star Count is wrong

### DISCUSSION THREAD

**Discussion by nilsocket** [community/community#23160] [New to GitHub]
<p>I have just removed all stars(Unstared) from Github, but the count remains 1.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/3/c36fa73102a46204848bdc2b92d8231c7f64e8c4.png" data-download-href="/uploads/short-url/rSUjZeU9qXzygAzhzh7k0L07leA.png?dl=1" title="Github_Bug" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/29815708/181075390-8225c30a-1c06-4fdd-a857-a51f39beebfe.png" alt="Github_Bug" data-base62-sha1="rSUjZeU9qXzygAzhzh7k0L07leA" width="690" height="340" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/3/c36fa73102a46204848bdc2b92d8231c7f64e8c4_2_690x340.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/3/c36fa73102a46204848bdc2b92d8231c7f64e8c4_2_1035x510.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/3/c36fa73102a46204848bdc2b92d8231c7f64e8c4_2_1380x680.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/3/c36fa73102a46204848bdc2b92d8231c7f64e8c4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Github_Bug</span><span class="informations">1392×686 55.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>The star count includes repositories and topics that are starred. As you can see in the screenshot, you’ve starred the “Go” topic, so that counts as 1.</p>
